platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/views/LearnerPushMerkleTreeView.java

```diff
@@ -273,4 +275,26 @@ public void releaseNode(final MerkleNode node) {
             node.release();
         }
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void recordHashStats(
+            @NonNull final ReconnectMapStats mapStats,
+            @NonNull final MerkleNode parent,
+            final int childIndex,
+            final boolean nodeAlreadyPresent) {
+        assert parent.isInternal();
+        final MerkleNode child = parent.asInternal().getChild(childIndex);
+        // The child may be missing per the getChild() specification,
+        // and this method cannot reason about a `null`, so just bail.
+        if (child == null) return;
```

Comment:
is that allowed in our format guideline?

Reply:
I believe you either have a one-liner, or use a block. In this case, this is a one-liner, so it should be allowed.

Reply:
Please see https://www.notion.so/swirldslabs/007-Java-Code-Style-a0130a7059b044f68447b9ca3a5242b3?pvs=4#490693eca99841ed9071158baa8a4175